corehq/apps/api/odata/views.py

```diff
@@ -51,6 +51,45 @@ def get(self, request, domain):
         return add_odata_headers(HttpResponse(metadata, content_type='application/xml'))
 
 
+class ODataFormServiceView(View):
+
+    urlname = 'odata_form_service'
+
+    @method_decorator(basic_auth_or_try_api_key_auth)
+    @method_decorator(toggles.ODATA.required_decorator())
+    def get(self, request, domain, app_id):
+        data = {
+            '@odata.context': absolute_reverse(ODataFormMetadataView.urlname, args=[domain, app_id]),
+            'value': [
+                {
+                    'name': xmlns,
+                    'kind': 'EntitySet',
+                    'url': xmlns,
+                }
+                for xmlns in get_xmlns_by_app(domain, app_id)
```

Comment:
Okay, I see what's going on with this `xmlns` stuff. I think maybe we can switch the whole thing to use `Form.unique_id` which will also solve the custom xmlns issue? Only downside (or is it an upside?) is that forms from different apps with then not have their data joined. This might be a divergence from exports.

Reply:
I think we want to use xmlns to maintain parity with exports.  We could use the full xmlns if we perform URL escaping to handle forward slashes.  I did these hacks to workaround the forward slashes, but escaping should do the trick.

Reply:
Cool, that seems like a good compromise

Reply:
On second thought, these URLs are going to be replaced when we switch to user configured feeds.  I think we can punt on this change - thoughts?

Reply:
I guess hard to say without knowing the rationale for punting. Is escaping hard to implement, or you're worried about UX, or something else?

Reply:
Rational is mostly that we won't be using this URL endpoint in the user-facing solution once Ethan's work is complete, so I don't want to spend time on something that won't be used except in internal testing.  The LOE is a bit more than an easy win since the %'s aren't handled well.

Reply:
Gotcha. If we're confident that update is imminent then I'm okay with the plan.